Danki2/Assets/Scripts/Effects/EffectManager.cs

```
@@ -57,6 +62,8 @@ public void AddActiveEffect(Effect effect, float duration)
     /// <returns> The Guid to use to remove the effect. </returns>
     public Guid AddPassiveEffect(Effect effect)
     {
+        if (actor.Dead) return default;
```

Returning default here could be a bit misleading, could make this method `TryAddPassiveEffect` instead with an `out`?

---

done